<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QMainWindow" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>607</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time (s):</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>171</width>
      <height>411</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_voltage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage (V):</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot_2" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Torque</string>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="plot_3" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Orientation</string>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="plot_4" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>EMG</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Exo</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>EMG</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxBaudrates">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>X-IMU</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Com:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxPortNames</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxBaudRates"/>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBoxPortNames"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>&amp;Baudrate:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxBaudRates</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButtonOnOff">
       <property name="text">
        <string>open</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="textArea">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textArea_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>380</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxBaudrates_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>290</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Nano</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButtonOnOff_2">
       <property name="text">
        <string>open</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>&amp;Com:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxPortNames</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>&amp;Baudrate:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxBaudRates</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxBaudRates_2"/>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBoxPortNames_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_start</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>on_btn_start_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_stop</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>on_btn_stop_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_reset</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>on_btn_reset_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>814</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_btn_start_clicked()</slot>
  <slot>on_btn_stop_clicked()</slot>
  <slot>on_btn_reset_clicked()</slot>
 </slots>
</ui>
